Sources/MuxPlayerSwift/PublicAPI/Options/PlaybackOptions.swift

```diff
@@ -197,16 +203,19 @@ extension PlaybackOptions {
         )
     }
 
-    /// Initializes playback options with a
+    /// Initializes playback options for use with Mux Video DRM
```

Comment:
We probably still want to keep exposed an initializer that only takes a `playbackToken` for supporting signed playback URLs without DRM.

Reply:
oops! agreed for sure